assignment-3.md

```diff
@@ -0,0 +1,108 @@
+# Assignment 3: Tests
+
+## 3.1 Getting Started
+
+Start the test runner with `npm test`.
+
+By default only tests for changed files are run. So in case you have been making
+Git commits, and currently have no uncommitted changes in your working
+directory, press `a` to trigger a test run.
+
+The initial test in `Climate.test.tsx` probably does not run green anymore,
+because it was testing the app in its starting state, but we have changed the
+app quite a bit in the last exercises.
+
+We'll take care for that soon, but for now disable that test with Jest's `skip`
+feature. Change this:
+
+```typescript
+test('Climate app shows the current temperature…', …)
+```
+
+to this:
+
+```typescript
+test.skip('Climate app shows the current temperature…', …)
+```
+
+## 3.2 Header
+
+Let's get started by writing some tests for the `Header` component that we
+have implemented in the first assignment. You have probably put that into a
+file called `Header.tsx`. Create a new file `Header.test.tsx` for the tests.
```

Comment:
maybe add a remark that says "by convention the test files have the same name as the component file"
if this is not mentioned in th slides.

Reply:
I'll put a bullet point for the on the slides. Here I think it should be implicitly clear,
as it says "you have `Header.tsx`, create `Header.test.tsx`", I think everybody should
get from that to also use this naming for further test files.

Reply:
Tracking with issue 65 in our slides board, closing this thread.